cgroups/fs/hugetlb.go

```diff
@@ -25,5 +40,37 @@ func (s *HugetlbGroup) Remove(d *data) error {
 }
 
 func (s *HugetlbGroup) GetStats(path string, stats *cgroups.Stats) error {
+	pageSizes, err := getHugePageSize()
```

Comment:
@vmarmol  Any XXXGroup is created When subsystems is created. And in Manager.Apply() used first. Do you mean put it into Manager.Apply()?